lib/explorer/polars_backend/expression.ex

```diff
@@ -172,6 +173,11 @@ defmodule Explorer.PolarsBackend.Expression do
     raise ArgumentError, "missing #{inspect(__MODULE__)} nodes: #{inspect(missing)}"
   end
 
+  def to_expr(%LazySeries{op: :from_json, args: _}) do
```

Comment:
```suggestion
  def to_expr(%LazySeries{op: :from_json}) do
```

@josevalim If i drop `args: _` i get a lot of warnings even though all tests pass. Why does this happen ?
<details>

<summary>Log containing warnings</summary>
```
09:59 $ mix ci
    Finished dev [unoptimized + debuginfo] target(s) in 0.42s
Compiling 3 files (.ex)
     warning: this clause cannot match because a previous clause at line 265 always matches
     │
 314 │     def to_expr(%LazySeries{op: unquote(op), args: unquote(args)}) do
     │     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     │
     └─ lib/explorer/polars_backend/expression.ex:314

     warning: this clause cannot match because a previous clause at line 265 always matches
     │
 314 │     def to_expr(%LazySeries{op: unquote(op), args: unquote(args)}) do
     │     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     │
     └─ lib/explorer/polars_backend/expression.ex:314

     warning: this clause cannot match because a previous clause at line 265 always matches
     │
 314 │     def to_expr(%LazySeries{op: unquote(op), args: unquote(args)}) do
     │     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     │
     └─ lib/explorer/polars_backend/expression.ex:314

     warning: this clause cannot match because a previous clause at line 265 always matches
     │
 314 │     def to_expr(%LazySeries{op: unquote(op), args: unquote(args)}) do
     │     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     │
     └─ lib/explorer/polars_backend/expression.ex:314

     warning: this clause cannot match because a previous clause at line 265 always matches
     │
 314 │     def to_expr(%LazySeries{op: unquote(op), args: unquote(args)}) do
     │     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     │
     └─ lib/explorer/polars_backend/expression.ex:314

     warning: this clause cannot match because a previous clause at line 265 always matches
     │
 314 │     def to_expr(%LazySeries{op: unquote(op), args: unquote(args)}) do
     │     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     │
     └─ lib/explorer/polars_backend/expression.ex:314

     warning: this clause cannot match because a previous clause at line 265 always matches
     │
 314 │     def to_expr(%LazySeries{op: unquote(op), args: unquote(args)}) do
     │     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     │
     └─ lib/explorer/polars_backend/expression.ex:314

     warning: this clause cannot match because a previous clause at line 265 always matches
     │
 314 │     def to_expr(%LazySeries{op: unquote(op), args: unquote(args)}) do
     │     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     │
     └─ lib/explorer/polars_backend/expression.ex:314

     warning: this clause cannot match because a previous clause at line 265 always matches
     │
 314 │     def to_expr(%LazySeries{op: unquote(op), args: unquote(args)}) do
     │     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     │
     └─ lib/explorer/polars_backend/expression.ex:314

     warning: this clause cannot match because a previous clause at line 265 always matches
     │
 314 │     def to_expr(%LazySeries{op: unquote(op), args: unquote(args)}) do
     │     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     │
     └─ lib/explorer/polars_backend/expression.ex:314

Compiling crate explorer in debug mode (native/explorer)
   Compiling explorer v0.1.0 (/Users/kartheek/erlang-ws/github-ws/latest/explorer/native/explorer)
    Finished dev [unoptimized + debuginfo] target(s) in 5.26s
Excluding tags: [:cloud_integration]

.................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................*................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
Finished in 2.8 seconds (2.8s async, 0.00s sync)
582 doctests, 1390 tests, 0 failures, 18 excluded, 1 skipped
</details>